src/css/application.css

```diff
@@ -133,72 +139,96 @@ body {
 }
 
 .top-bar__logo-container > svg {
-  height: 3em;
+  width: 3em;
 }
 
 .top-bar__wordmark-container {
-  position: relative;
-  margin-left: -0.25em;
+  margin: 0 1em 0 0.5em;
 }
 
 .top-bar__wordmark-container > svg {
-  padding: 0 1em;
-  max-height: 1em;
-  max-width: 100%;
-  width: auto;
+  width: 6.5em;
 }
 
 .top-bar__snapshot_in-progress {
-  color: var(--color-light-gray);
+  opacity: 0.5;
   cursor: progress;
 }
 
 .top-bar__avatar {
-  height: 2em;
-  border-radius: 50%;
+  border-radius: var(--rounded-corners);
+  height: 1.8em;
   vertical-align: middle;
-  line-height: 1em;
 }
 
 .top-bar__username {
   margin-left: 0.5em;
 }
 
 .top-bar__menu-button {
-  align-self: stretch;
-  border-color: transparent;
-  border-style: solid;
-  border-width: 0 1px;
+  border-radius: var(--rounded-corners);
   box-sizing: border-box;
   cursor: pointer;
-  line-height: 3em;
-  padding: 0 0.5em;
+  height: 2.8em;
+  line-height: 2.8em;
+  padding: 0 1em;
   position: relative;
   user-select: none;
   vertical-align: middle;
+  margin-right: 0.5em;
+  white-space: nowrap;
 }
 
-.top-bar__menu-button_active {
-  background-color: white;
-  border-color: var(--color-low-contrast-gray);
+.top-bar__menu-button:last-child {
+  margin-right: 0;
+}
+
+.top-bar__menu-button:hover {
+  box-shadow: 0 0.1em 0.2em 0 rgba(0, 0, 0, 0.3);
+}
+
+.top-bar__menu-button_primary,
+.top-bar__menu-button_secondary {
+  box-shadow: 0 0 0.2em 0 rgba(0, 0, 0, 0.3);
+}
+
+.top-bar__menu-button_primary {
+  background-color: rgba(255, 255, 255, 0.5);
+}
+
+.top-bar__menu-button_active,
+.top-bar__menu-button_active:hover,
+.top-bar__menu-button:active {
+  box-shadow: inset 0 0.1em 0.2em 0 rgba(0, 0, 0, 0.3);
 }
 
 .top-bar__hamburger {
-  font-size: 2em;
+  font-size: 1.5em;
 }
 
 .top-bar__menu {
-  border: 1px solid var(--color-low-contrast-gray);
-  box-shadow: 2px 2px 2px var(--color-low-contrast-gray);
-  left: -1px;
+  border-radius: var(--rounded-corners);
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.25);
   position: absolute;
-  top: 100%;
+  top: 3.4em;
```

Comment:
So it makes sense why this can’t be 100%—I had forgotten that it is a child of the button and not the `TopBar` directly— but I still don’t love duplicating the knowledge of how tall overall the top bar is going to be. Can we extract that information into variables and/or use a `calc` statement to make more explicit how we get to `3.4em` here?